airflow-core/src/airflow/ui/public/i18n/locales/de/dags.json

```
@@ -34,6 +34,10 @@
       "error": "Fehler beim Zurücksetzen von {{type}}",
       "title": "{{type}} bereinigen und neu planen"
     },
+    "confirmationDialog": {
+      "description": "Der Task befindet sich derzeit in dem Zustand {{state}}, ausgelöst vom Benutzer {{user}} um {{time}}. \nDer Task kann nicht zurückgesetzt werden, bis er abgeschlossen ist, oder die Option \"Zurücksetzen von laufenden Tasks verhindern\" darf im Dialogfeld zum Zurücksetzen des Tasks nicht genutzt werden.",
```

```suggestion
      "description": "Der Task befindet sich derzeit in dem Zustand {{state}}, ausgelöst durch den Benutzer {{user}} um {{time}}. \nDer Task kann nicht zurückgesetzt werden, bevor er abgeschlossen ist, oder ein Benutzer im Zurücksetz-Dialog die Option \"Zurücksetzen von laufenden Tasks verhindern\" deaktiviert.",
```

Kleiner Vorschlag :)

---

Danke! Das ist noch etwas klarer, was gemeint ist!